src/translations/generate_translations.ts

```diff
@@ -30,13 +30,20 @@ for (var i = 0; i < workbook.SheetNames.length; i++) {
       // row[3]: key to use in application
       const key = row[3];
 
-      if (!es) {
-        console.log("spanish translation missing for ", row[0]);
+      if (!key) {
```

Comment:
it feels ok to pull in not-yet-ready content, b/c then we can also immediately work out of the sheet while developing. also if there are cracks in our translation process, seems slightly better to accidentally merge an english fallback than to merge w/ no translations in (fwiw i checked google analytics and we've only had 3 spanish users since the site launched, far less than 1% of traffic). but no strong feelings here

Reply:
skipping content that doesn't have the key doesn't mean you can't work directly out of the sheet while developing; just means the key "undefined" won't be added to the sheet so you can avoid pulling in work-in-progress content to avoid more conflict resolutions in the translations json documents